Dockerfile

```diff
@@ -1,15 +1,35 @@
-FROM circleci/python:3.7-node
+FROM python:3.7
+
+# FIXME: switch to non-root user
 USER root
-RUN mkdir -p /srv/code
-WORKDIR /srv/code
-#Copy code from current directory to /srv/code/ in image
-ADD . .
-# Install Python dependencies
-RUN pipenv install --system --deploy
-#Install static dependencies
-RUN npm i
-
-EXPOSE 80
-
-RUN npm rebuild node-sass
-RUN npm run build
\ No newline at end of file
+
+# Create app working directory
+RUN mkdir -p /app
+WORKDIR /app
+
+# Install system dependencies and target node v14
+RUN curl -fsSL https://deb.nodesource.com/setup_14.x | bash - && \
+    apt-get update && \
+    apt-get install -y netcat git nodejs && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install python dependencies
+COPY ./Pipfile.lock ./Pipfile /app/
+RUN pip install pipenv && \
+    pipenv install --system --deploy && \
```

Comment:
I'm still a little confused about what `--deploy` does. I see this:

> `--deploy` — Make sure the packages are properly locked in Pipfile.lock, and abort if the lock file is out-of-date.
https://pipenv-fork.readthedocs.io/en/latest/basics.html

> This will fail a build if the Pipfile.lock is out–of–date, instead of generating a new one. Or you can install packages exactly as specified in Pipfile.lock using the `sync` command.
https://pipenv-fork.readthedocs.io/en/latest/advanced.html#using-pipenv-for-deployments